carma_cooperative_perception/README.md

```diff
@@ -0,0 +1,32 @@
+# CARMA cooperative perception stack
+
+The `carma_cooperative_perception` package contains the cooperative perception stack used in CARMA Platform.
+It wraps the `cooperative_perception_core` library in a ROS 2 interface. The stack provides two main functionalities.
+The multiple object tracking functionality aggregates object data from incoming Sensor Data Sharing Messages (SDSMs),
+Basic Safety Messages (BSMs), and local sensors. The sensor data sharing functionality broadcasts SDSMs containing
```

Comment:
BSMs will likely not be included in the initial implementation. Did you already finish the vehicle side implementation for BSM to detection?